sdk/python/feast/feature_server.py

```diff
@@ -339,8 +339,8 @@ def start_server(
     workers: int,
     keep_alive_timeout: int,
     registry_ttl_sec: int,
-    ssl_key_path: str,
-    ssl_cert_path: str,
+    tls_key_path: str,
+    tls_cert_path: str,
     metrics: bool,
 ):
     if metrics:
```

Comment:
I think its a good idea to do
if (tls_key_path and not tls_cert_path) or (not tls_key_path and tls_cert_path)
as well